packages/flutter/lib/src/gestures/mouse_tracking.dart

```diff
@@ -131,8 +131,8 @@ class MouseTracker extends ChangeNotifier {
   /// Stops tracking an annotation, indicating that it has been removed from the
   /// layer tree.
   ///
-  /// If the associated layer is not removed, and receives a hit, then
-  /// [sendMouseNotifications] will assert the next time it is called.
+  /// If the associated layer is not removed, then an assertion error will be
```

Comment:
This sentence is a bit confusing, do you mean `An assertion error will be thrown if the associated layer is not removed and receives another mouse hit`

Reply:
Thanks! Fixed it in your way